### PR TITLE
Deprecate Brew Evolution.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
This repository hasn't really served it's purpose as not all proposals have been implemented or sufficiently prioritised. There's been
insufficient community interest in this process and it's much heavier than discussion on an issue. Additionally, it can provide a "set in
stone" feeling where design has been agreed but there were unforeseen problems with the implementation.

### CODE REVIEW

**Comment by scpeters** [comment]
It was worth a try.

We should probably update the issue template in brew that currently points to this repository for feature requests:

~~~
### Propose a feature:

Instead of creating an issue here, please create a pull request with your change proposal in the
[Homebrew Evolution](https://github.com/Homebrew/brew-evolution) repository using the
[proposal template](https://github.com/Homebrew/brew-evolution/blob/master/proposal_template.md).
~~~

**Comment by MikeMcQuaid** [comment]
@scpeters I have branches for that locally that will be pushed up shortly.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1 +1,3 @@
-Please use the [proposal template](/proposal_template.md) for your pull request.
+**PLEASE DO NOT CREATE NEW PULL REQUESTS ON THIS REPOSITORY**
+
+This repository was an experiment to tracks the ongoing evolution of Homebrew. It has been replaced with [feature requests on Homebrew/brew](https://github.com/Homebrew/brew/issues/new).
```
Comment (vitorgalvao) [inline .github/PULL_REQUEST_TEMPLATE.md]: Typo. “tracks” should be “track”.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,2 @@
...
-This repository tracks the ongoing evolution of Homebrew. It contains:
-
-* The [Homebrew evolution process](process.md) that governs the evolution of Homebrew.
-* [Commonly Rejected Proposals](commonly_rejected_proposals.md) which have been denied in the past.
-
----
-
-Inspired by the [Swift Programming Language Evolution](https://github.com/apple/swift-evolution) repository.
+# Homebrew Evolution (deprecated)
+This repository was an experiment to tracks the ongoing evolution of Homebrew. It has been replaced with [feature requests on Homebrew/brew](https://github.com/Homebrew/brew/issues/new).
```
Comment (vitorgalvao) [inline README.md]: Same here.

**Pull request merged**